src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -11,7 +11,7 @@
 
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
-            <argument type="service" id="service_container" />
+            <argument /> <!-- fragment renderer locator -->
```

Comment:
Sadly it looks like getter injection on abstract classes fail if the service is an event listener:

> Error: Cannot instantiate abstract class Symfony\Component\HttpKernel\EventListener\SessionListener
>
> /Volumes/HD/symfony/symfony/src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php:109